exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -436,143 +542,336 @@ func (e *Exchange) GetWithdrawalsHistory(ctx context.Context, c currency.Code, _
 
 // GetRecentTrades returns the most recent trades for a currency and asset
 func (e *Exchange) GetRecentTrades(ctx context.Context, pair currency.Pair, assetType asset.Item) ([]trade.Data, error) {
-	return e.GetHistoricTrades(ctx, pair, assetType, time.Now().Add(-time.Minute*15), time.Now())
+	fPair, err := e.FormatExchangeCurrency(pair, assetType)
+	if err != nil {
+		return nil, err
+	}
+
+	var resp []trade.Data
+	switch assetType {
+	case asset.Spot:
+		var tradeData []*Trade
+		tradeData, err = e.GetTrades(ctx, fPair, 0)
```

Comment:
🚧 Change request:
* 🌴 I raised this before, but since it's happening a lot let's fix it everywhere in this PR please
```suggestion
		tradeData, err := e.GetTrades(ctx, fPair, 0)
```